closed/autoconf/custom-hook.m4

```diff
@@ -340,6 +341,25 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_OJDKTHREAD_SUPPORT],
+[
+  AC_MSG_CHECKING([for OpenJDK Thread support])
+  AC_ARG_ENABLE([ojdkthread-support], [AS_HELP_STRING([--enable-ojdkthread-support], [enable OpenJDK Thread support @<:@disabled@:>@])])
+  OPENJ9_ENABLE_OJDKTHREAD_SUPPORT=false
```

Comment:
Can we use `OPENJ9_ENABLE_OPENJDK_THREAD_SUPPORT` instead of `OPENJ9_ENABLE_OJDKTHREAD_SUPPORT`, please? This would be consistent with `OPENJ9_ENABLE_OPENJDK_METHODHANDLES` we had when migrating the method handle implementation.

Ideally, the OpenJ9 JPP configuration flag should also not be so abbreviated.

Reply:
There weren't much precedents for migrating RI implementation requiring additional configuration options. Out of respect the naming `OPENJ9_ENABLE_OPENJDK_METHODHANDLES`,  `OPENJ9_ENABLE_OJDKTHREAD_SUPPORT` seems concise than `OPENJ9_ENABLE_OPENJDK_THREAD_SUPPORT` while avoiding double `OPEN`s. Along with JPP flag `OJDKTHREAD_SUPPORT`, these are also based on `--enable-ojdkthread-support` proposed by @tajila .

Reply:
The project is called 'openjdk' we should not be mangling that name (just look at the name of this repo for reinforcement).

I agree the configure option name should be consistent with the macro name, so in summary, the option should be `--enable-openjdk-thread-support` and the macro should be `OPENJ9_ENABLE_OPENJDK_THREAD_SUPPORT`.

BTW, the two 'open' substrings refer to different projects (openj9 vs. openjdk).

Reply:
renamed.